connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/WorkerGroupMember.java

```diff
@@ -223,4 +235,12 @@ private void stop(boolean swallowException) {
         else
             log.debug("The Connect group member has stopped.");
     }
+
+    /**
+     * Method for unit tests
+     * @return
+     */
+    public Metrics getMetrics() {
+        return this.metrics;
+    }
```

Comment:
Nit: this doesn't follow our conventions: we don't use the `get*` style getters, and if this is only used in unit tests we should make this package protected instead this should be:
```suggestion
    // Visible for testing
    Metrics metrics() {
        return this.metrics;
    }
```
You'll have to change the `WorkerGroupMemberTest` accordingly.

Reply:
Renamed this method to metrics() and make it protected.